cmd/manifest/cmd/create.go

```diff
@@ -89,23 +90,23 @@ func create(cmd *cobra.Command, args []string) {
 	}
 	digestBytes := sha256.Sum256(firmwareBytes)
 
-	gitTagName, err := semver.NewVersion(gitTag)
+	gitSemVer, err := semver.NewVersion(strings.TrimPrefix(gitTag, "v"))
 	if err != nil {
 		log.Fatalf("Failed to parse git_tag: %v", err)
 	}
-	tamagoVersionName, err := semver.NewVersion(tamagoVersion)
+	tamagoSemVer, err := semver.NewVersion(strings.TrimPrefix(tamagoVersion, "v"))
```

Comment:
Is this `TrimPrefix` just for insurance, or is this needed? i.e. I don't think we pass it in beginning with a `v` right now?

Reply:
That's right, the way it's currently called does not include the "v". I added it for consistency, if you were ever using the manifest tool outside of our pipelines. I can remove it, if you prefer?

Reply:
I think it's fine, I just wanted to make sure my understanding of what was needed and what was insurance was good. Ta!